### ISSUE TITLE
Backdrop's PHP version requirement prevents it from being installed on Bluehost

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2967]
## Describe your issue or idea

When trying to deploy a backdrop site to Bluehost, I encountered this error: 
`Your PHP installation is too old. Backdrop CMS requires at least PHP 5.3.2. See the System Requirements page for more information.`

This is because Bluehost ships with PHP Version 5.2.17. Drupal 7's minimum PHP version is 5.2.4, meaning Drupal 7 can be installed on Bluehost. 

Is there a reason Backdrop can't match Drupal 7's minimum PHP version requirements? What are we using in php 5.3.2?

**Comment by opi** [comment]
Some food: 
- Older supported version of PHP is 5.6 https://secure.php.net/supported-versions.php
- Debian Jessie (old stable) PHP version is 5.6 https://packages.debian.org/fr/jessie/php5

**Comment by findlabnet** [comment]
Haven't used this hosting, but:
> Our servers support PHP 5.4, PHP 5.6, and PHP 7.0.

 https://my.bluehost.com/hosting/help/447

**Comment by jlfranklin** [comment]
> Is there a reason Backdrop can't match Drupal 7's minimum PHP version requirements? What are we using in php 5.3.2?

IIRC, PHP 5.3 significantly improved the performance of code using objects. 

Looking at the release notes, PHP 5.3 added [INTL by default](http://php.net/manual/en/migration53.new-extensions.php), but that's not required to run Backdrop.  Do we use any of the [new functions in PHP 5.3](http://php.net/manual/en/migration53.functions.php)?  I thought we used some of the new [DateTime methods](http://php.net/manual/en/migration53.methods.php), but I couldn't find an example quickly.  Other changes are listed [here](http://php.net/releases/5_3_0.php). (Lambda functions?)

Even if it doesn't break core, there are likely contrib modules that will break by backing up to 5.2.

Given how old PHP 5.2 is, I'm surprised any hosting company still offers it at all, even as a "support for really old software" compatibility option.

**Comment by olafgrabienski** [comment]
>> This is because Bluehost ships with PHP Version 5.2.17.

>> Our servers support PHP 5.4, PHP 5.6, and PHP 7.0.

> considering this was the second shared host I'd tried to put Backdrop onto and encountered the problem

@jenlampton Doesn't Bluehost provide an option to switch the PHP version?

**Comment by jenlampton** [comment]
@olafgrabienski yes. It does, and that's what I eventually did. I was just wondering what it was about Backdrop that bumped its PHP version requirement above that of Drupal 7.

After chatting with @quicksketch we've determined it's the Date module we merged into core. I Drupal 7 there was a "Date PHP 5.2" module that needed to be enabled to use dates on older versions of PHP. IMHO what we've done in Backdrop is worth the bump to PHP.5.3.2 :)  Thanks all for your feedback, I'm going to close this issue (unless anyone objects).
